src/datanode/src/heartbeat/handler.rs

```diff
@@ -46,7 +47,7 @@ impl RegionHeartbeatResponseHandler {
             Instruction::OpenRegion(region_ident) => {
                 let region_id = Self::region_ident_to_region_id(&region_ident);
                 let open_region_req = RegionRequest::Open(RegionOpenRequest {
-                    engine: region_ident.table_ident.engine,
+                    engine: default_engine().to_string(),
```

Comment:
`RegionIdent` should contain the engine field.

Reply:
@waynexia Can the field `engine` in `RegionOpenRequest` be removed? Don't see the actual usage.

Reply:
> @waynexia Can the field `engine` in `RegionOpenRequest` be removed? Don't see the actual usage.

Datanode needs to know which engine to use to open the region: https://github.com/GreptimeTeam/greptimedb/blob/develop/src/datanode/src/region_server.rs#L207